py/server/deephaven/experimental/iceberg.py

```diff
@@ -243,6 +340,95 @@ def table(self, instructions: Optional[IcebergInstructions] = None, snapshot_id:
             return IcebergTable(self.j_object.table(snapshot_id, instructions))
         return IcebergTable(self.j_object.table(instructions))
 
+    def append(self,
+               table_identifier: str,
+               tables: List[Table],
+               partition_paths: Optional[List[str]] = None,
```

Comment:
```suggestion
               partition_paths: Optional[Sequence[str]] = None,
```
Confirm with @jmao-denver